### ISSUE TITLE
Classifier: Design Mock Classifier Architecture

### ISSUE THREAD

**Issue by shelabd** [buerokratt/Mock-Classifier#1]

**Comment by martinkearn** [comment]
Agreed on 27/04 stand-up that we will build a mock for this project. The work on the Cognitive Language service version of the Classifier will be undertaken in parallel with Veronika Mugra and not tracked here.

**Comment by martinkearn** [comment]
Agreed on 28/04 stand-up with @turnerrainer that we'll set the Classifier up to use English language utterances and ministry names (not Estonian)
